--- UHDDevice.cpp	2018-02-10 10:50:23.397600596 -0500
+++ /tmp/UHDDevice.cpp	2018-02-10 10:26:44.770357681 -0500
@@ -82,11 +82,12 @@
 	{ B2XX,  99 },
 	{ X300,  73 },
 	{ UMTRX, 0 },
+	{ LimeSDRMini, 5 },
 };
 
 static int get_dev_offset(enum uhd_dev_type type)
 {
-	if ((type != B2XX) && (type != USRP2) && (type != X300)) {
+	if ((type != B2XX) && (type != USRP2) && (type != X300) && (type != LimeSDRMini)) {
 		LOG(ALERT) << "Unsupported device type";
 		return 0;
 	}
@@ -259,7 +260,7 @@
 {
 	std::string mboard_str, dev_str;
 	uhd::property_tree::sptr prop_tree;
-	size_t usrp2_str, b200_str, b210_str, x300_str, x310_str;
+	size_t usrp2_str, b200_str, b210_str, x300_str, x310_str, LimeSDRMini_str;
 
 	prop_tree = usrp_dev->get_device()->get_tree();
 	dev_str = prop_tree->access<std::string>("/name").get();
@@ -270,6 +271,8 @@
 	b210_str = mboard_str.find("B210");
 	x300_str = mboard_str.find("X300");
 	x310_str = mboard_str.find("X310");
+	LimeSDRMini_str = mboard_str.find("LimeSDR-Mini");
+LOG(ALERT) << "found: " << mboard_str << std::endl;
 
 	if (b200_str != std::string::npos) {
 		dev_type = B2XX;
@@ -281,6 +284,8 @@
 		dev_type = X300;
 	} else if (x310_str != std::string::npos) {
 		dev_type = X300;
+	} else if (LimeSDRMini_str != std::string::npos) {
+		dev_type = LimeSDRMini;
 	} else {
 		goto nosupport;
 	}

